src/engine-scripts/puppet/onReady.js

```diff
@@ -25,26 +25,24 @@ module.exports = async ( page, scenario ) => {
 	if ( hashtags.includes( '#vector-2022' ) ) {
 		await require( './mainMenuState' )( page, hashtags );
 		await require( './userMenuState' )( page, hashtags );
-		await require( './collapsedTocState' )( page, hashtags );
-	}
+		await require( './unpinnedTocState' )( page, hashtags );
 
-	if ( hashtags.includes( '#toggle-toc-subsections' ) ) {
-		await require( './toggleTocSubsections' )( page );
+		if ( hashtags.includes( '#toggle-toc-subsections' ) ) {
+			await require( './toggleTocSubsections' )( page );
+		}
+		if ( hashtags.includes( '#echo' ) ) {
```

Comment:
The [echo report](https://pixel.wmcloud.org/reports/echo/index.html) tests both the vector-2022 and minerva skin so we should probably keep this outside the conditional

Reply:
Also, CI is failing because of some eslint issues